cancermuts/datasources.py

```diff
@@ -1601,10 +1602,24 @@ def add_mutations(self, sequence, metadata=[]):
         if self._df is None:
             return
 
+        #Prepares metadata for mutations
+        do_genomic_mutations=False
+        if 'genomic_mutations' in metadata:
+            do_genomic_mutations=True
+
+        if do_genomic_mutations:
+            gm_df = self._df['genomic_mutations'][self._df['type'] == 'mutation']
+            gm = list()
+            for row in gm_df:
+                if row is not '':
```

Comment:
I realised that this should use the equality test operator `==` rather than `is`, which is used for object equality. Python also warns us:

```
In [1]: a = ''

In [2]: a is ''
<>:1: SyntaxWarning: "is" with a literal. Did you mean "=="?
<ipython-input-2-defaa20e97c9>:1: SyntaxWarning: "is" with a literal. Did you mean "=="?
  a is ''
Out[2]: True
```